File: guides/micronaut-openapi-swagger-ui/java/src/main/java/example/micronaut/HomeController.java
package example.micronaut;

import io.micronaut.http.HttpResponse;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;
import io.micronaut.http.uri.UriBuilder;
import io.swagger.v3.oas.annotations.Hidden;
import java.net.URI;

@Controller // <1>
class HomeController {

    private final static URI SWAGGER_UI = UriBuilder.of("/swagger-ui").path("index.html").build();

    @Get // <2>
    @Hidden // <3>
    HttpResponse<?> home() {
        return HttpResponse.seeOther(SWAGGER_UI);
    }
}

File: guides/micronaut-openapi-swagger-ui/python/src/example/micronaut/home_controller.py
import java

from micronaut.http import HttpResponse
from micronaut.http.annotation import Controller, Get
from micronaut.http.uri import UriBuilder


Hidden = java.type("io.swagger.v3.oas.annotations.Hidden")


@Controller  # <1>
class HomeController:

    SWAGGER_UI = UriBuilder.of("/swagger-ui").path("index.html").build()

    @Get  # <2>
    @Hidden  # <3>
    def home(self) -> HttpResponse:
        return HttpResponse.seeOther(self.SWAGGER_UI)
